roles/kubernetes/control-plane/defaults/main/main.yml

```diff
@@ -236,3 +236,8 @@ kube_apiserver_tracing_sampling_rate_per_million: 100
 
 # Enable kubeadm file discovery if anonymous access has been removed
 kubeadm_use_file_discovery: "{{ remove_anonymous_access }}"
+
+# Supported asymmetric encryption algorithm types for the cluster's keys and certificates.
+# can be one of RSA-2048(default), RSA-3072, RSA-4096, ECDSA-P256
```

Comment:
The upstream docs apparently disagree:
```
encryptionAlgorithm holds the type of asymmetric encryption algorithm used for keys and certificates. Can be "RSA" (default algorithm, key size is 2048) or "ECDSA" (uses the P-256 elliptic curve).
```
But given kubernetes/kubernetes#123054 I assume the docs are outdated